MAINTAINING.md

```diff
@@ -13,18 +13,39 @@ All operations are automated as much as possible.
 
 To update versions of images following steps are required:
 
-1.  Update all required versions in `Makefile`.
-2.  Update all required versions in `README.md`.
-3.  If you need to modify some `Dockerfile`s then do it via editing
-    [`templates/Dockerfile.erb` template](templates/Dockerfile.erb).
+1.  Update all required versions in `Makefile`. See also: [Image versioning convensions](#image-versioning-convensions)
+2.  Update all required versions in `README.md`. See also: [Image versioning convensions](#image-versioning-convensions)
+3.  Edit templates
+    1.  If you need to modify some `Dockerfile`s then do it via editing
+        [`templates/Dockerfile.erb` template](templates/Dockerfile.erb).
+    2.  If you need to modify some `Gemfile`s then do it via editing
+        [`templates/Gemfile.erb` template](templates/Gemfile.erb).
+    3.  If you need to modify some `conf`s then do it via editing
+        [`templates/conf` templates](templates/conf/)
 4.  Regenerate all `Dockerfile`s and their context (it's okay to remove previous
     ones completely):
     ```bash
     make src-all
     ```
-6.  Push changes to `master` branch.
+5.  Push changes to `master` branch.
 
+### Image versioning convensions
 
+1. We use the following image versioning convension:
+    1. `v<Fluentd version>-debian-<destination name>-<image major version>.<image minor version>`
+    2. Reset image version to `1.0` when bump up Fluentd version
+    3. Bump up image major version when including breaking changes on a destination
+    4. Bump up image minor version when updating gems on a destination
+
+**Note:** This procedure requests that the specified Fluentd debian tag image is already published in [Fluentd DockeHub's tags page](https://hub.docker.com/r/fluent/fluentd/tags).
+
+## Publish Images
+
+**Note:** This procedure requires `fluent/fluentd-kubernetes-daemonset` repository's DockerHub `Admin` privileges.
+
+Go to [Build settings page](https://hub.docker.com/repository/docker/fluent/fluentd-kubernetes-daemonset/builds) and then, push `[Trigger ▷]` buttons.
```

Comment:
It the description at the head of this file outdated?

> * Images and description on Docker Hub will be automatically rebuilt on pushes to master branch and on updates of parent Docker images.

Reply:
Oh. It is already outdated.

Reply:
Removed in https://github.com/fluent/fluentd-kubernetes-daemonset/pull/552/commits/f7df71a2ebeb9308337896519d1e762b8e3d7810.